src/safeposix/syscalls/net_calls.rs

```diff
@@ -1865,12 +1935,23 @@ impl Cage {
                                 );
                             }
 
-                            // simple if it's a domain socket
+                            //If the given socket is a Unix socket, lind handles
+                            //the connection, return with success
                             if sockhandle.domain == AF_UNIX {
```

Comment:
I think if were going to actually add backlog in for INET its worth putting a TODO tag here saying implement backlog for unix